EIPS/eip-ens-as-holder

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 5298
```
This was opened shortly after 5299, so in an abundance of caution to protect against EIP number sniping I'm assigning 5298 instead.

Reply:
Sounds good.